android/engine/src/test/java/org/smartregister/fhircore/engine/app/fakes/Faker.kt

```diff
@@ -90,4 +103,202 @@ object Faker {
       }
     }
   }
+
+  fun generatePatientRegisterConfiguration(): RegisterConfiguration {
```

Comment:
Is it possible to read the files from the asset directory instead of providing the Fakes?

Reply:
Yeah, that's possible